Resources/Prototypes/Entities/Stations/base.yml

```
@@ -78,6 +78,22 @@
 
+- type: entity
+  id: BaseStationShuttlesCargoOnly
+  abstract: true
+  components:
+    - type: StationCargoShuttle
+      path: /Maps/Shuttles/cargo.yml
+    - type: GridSpawn
+      groups:
+        trade:
+          addComponents:
+          - type: ProtectedGrid
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 12 but found 10

(indentation)